### ISSUE TITLE
Unimplemented Unicode escape in @

### ISSUE THREAD

**Issue by julesreid** [uiua-lang/uiua#88]
When using @ (character) to create a Unicode character that does not display directly, the output shows a syntax which includes the Unicode code point.  Running

```
+ 8192 @\0
```
gives:
```
@\u{2000}
```

If you then use the same syntax in the source, it is not recognized:

```
Error: Invalid escape character 'u'
   ╭─[:1:1]
 1 │ @\u{2000}
   │ ───  
───╯
```

Using the web page implementation of Uiua.

**Comment by kaikalii** [comment]
What was the character that you entered?
Inputing
```
@\0
```
in the web editor formats properly for me.

**Comment by julesreid** [comment]
OK, that figures.  The backslash syntax gave me the impression that it might work for input also.

**Comment by kaikalii** [comment]
That is my fault. I will make it clearer.
I may as well just make the output say `+8192@\0`.

**Comment by kaikalii** [comment]
Fixed in 38f86f7e0932e
